code/modules/speech/core.dm

```diff
@@ -157,7 +157,7 @@ Cleanup:
 
 Things To Implement:
 - AI
-- Virtual humans
+- Afterlife bar radio (:d for deadchat)
```

Comment:
Given that the deadchat PR is up, removing this may be best.

Reply:
*nod